transformation_service/.github/workflows/transformation_service_deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Deploy transformation_service to ECS'
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'transformation_service/**'
+env:
+  region: ${{ secrets.AWS_REGION }}
+  repository: ${{ secrets.AWS_REPOSITORY }}
+  serviceName: 'transformation_service'
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v1
+      - name: 'Login to Repository'
+        run: |
+          aws ecr get-login-password --region ${{ env.region }} | docker login --username AWS --password-stdin ${{ env.repository }}/${{ env.serviceName }}
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: 'Build Docker image'
+        run: |
+          docker build -t ${{ env.serviceName }} -f ./${{ env.serviceName }}/Dockerfile .
+      - name: 'Push Docker image to repository'
+        run: |
+          docker tag ${{ env.serviceName }}:latest ${{ env.repository }}/${{ env.serviceName }}:latest
+          docker push ${{ env.repository }}/${{ env.serviceName }}:latest
+      #- name: 'Redeploy to ECS'
```

Comment:
Ten redeploy je tu kvuli rychlemu nasazeni nove `latest` verze na ECS. Jde to zaridit i jinymi zpusoby, ale by default ECS zmeny v repozitari nesleduje (aspon myslim).

`--force-new-deployment` flag zajisti prave nasazeni nove verze.

Reply:
Já vím, ale jelikož zatím není hotový ECS, tak jsem to raději zakomentoval, aby to neházelo chyby. Odkomentuji za chvíli :-)